.circleci/config.yml

```diff
@@ -545,12 +545,6 @@ workflows:
             - build
           suite: itest-batch_deal
           target: "./itests/batch_deal_test.go"
-      - test:
-          name: test-itest-ccupgrade
-          requires:
-            - build
-          suite: itest-ccupgrade
-          target: "./itests/ccupgrade_test.go"
```

Comment:
a bit surprised t see t tests being removed 👁️

Reply:
This is the extremely replace-sector-with-another-sector CC upgrade, not snapdeals.

It's not even supported in actors anymore

Reply:
Woah